site/en/docs/webstore/cws-dashboard-listing/index.md

```diff
@@ -2,37 +2,54 @@
 layout: "layouts/doc-post.njk"
 title: "Complete your listing information"
 #date: TODO
-#updated: TODO
+updated: 2021-08-09
 description: How to add listing information for your Chrome Web Store item.
 ---
 
 {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/4RwZvdR4eACU46OCy7rm.png",
        alt="Screenshot of the Chrome Web Store developer dashboard item listing tab", height="472", width="800" %}
 
 Add the store listing information for your item that isn't in the ZIP file, such as a long
-description, screenshots, videos, and links to related sites. You'll need the following to finish
-your item's store listing:
+description, screenshots, videos and links to related sites. 
+
+## Product details 
+You'll need to fill out the following details about your item:
 
 - A **detailed description** of your item. Write your description to entice users to download your
   item. Make sure to start your description with a concise statement of what your item does, so
   users can understand the main idea and features at a glance. Your description can then go on to
   provide more detail, promotional copy, update logs, and so forth.
+  
+{% Aside 'caution' %}
+Be detailed but make sure your description complies with the [Keyword Spam][1] policy.
+{% endAside %}
+
+- The **primary [category][5]** where your item should be listed.
+- Your item's **language** to help users find it.
+
+## Graphic assets {: #graphic-assets }
+
+At a minimum, you should provide the following promotional images:
+
+- A 128x128 **store icon**.
 - At least one 1280x800 or 640x400 **screenshot** or YouTube **video** to show off what your item
   does. These are displayed in your item's Chrome Web Store listing in the following order:
   1.  Any localized video that you provide.
-  2.  Any localized screenshots that you provide.
-  3.  Any global (non-localized) video.
-  4.  Global screenshots.
+  1.  Any localized screenshots that you provide.
```

Comment:
Right on with the Markdown 1. 1. 1. list!

Reply:
yay!